qutip/core/states.py

```diff
@@ -51,7 +51,7 @@
 from .qobj import Qobj
 from .operators import jmat, displace
 from .tensor import tensor
-
+from .data import to
```

Comment:
Same comment about potential style - having looked through, I think that if `to.str2type` is private, then the import won't even be needed anyway.